src/python/sdk/examples/games.py

```diff
@@ -0,0 +1,440 @@
+                ECONIA_ADDR,
+                COIN_TYPE_EAPT,
+                COIN_TYPE_EUSDC,
+                market_id,
+                0,
+            )
+        )
+    except:
+        print("Account may already exist...")
+
+    try:
+        await client_b.gen_submit_tx_wait(
+            register_market_account(
+                ECONIA_ADDR,
+                COIN_TYPE_EAPT,
+                COIN_TYPE_EUSDC,
+                market_id,
+                0,
+            )
+        )
+        print("Initialized account pair...")
+    except:
+        print("Account may already exist...")
+
+    async def run():
+        flip = coin_flip()
+        client_maker = client_a if flip else client_b
+        client_taker = client_b if flip else client_a
+        flip = coin_flip()
+        from_type = COIN_TYPE_EAPT if flip else COIN_TYPE_EUSDC
+        to_type = COIN_TYPE_EUSDC if flip else COIN_TYPE_EAPT
+
+        base_lots_remaining = base_lots
+        if from_type == COIN_TYPE_EAPT:
+            await fund(
+                client_maker, client_maker.user_account, 100 * 10**8, COIN_TYPE_EAPT
+            )
+            await fund(
+                client_taker, client_taker.user_account, 600 * 10**6, COIN_TYPE_EUSDC
+            )
+
+            await client_maker.gen_submit_tx_wait(
+                deposit_from_coinstore(
+                    ECONIA_ADDR, COIN_TYPE_EAPT, market_id, 0, 100 * 10**8
+                )
+            )
+            await client_taker.gen_submit_tx_wait(
+                deposit_from_coinstore(
+                    ECONIA_ADDR, COIN_TYPE_EUSDC, market_id, 0, 600 * 10**6
+                )
+            )
+        else:
+            await fund(
+                client_maker, client_maker.user_account, 600 * 10**6, COIN_TYPE_EUSDC
+            )
+            await fund(
+                client_taker, client_taker.user_account, 100 * 10**8, COIN_TYPE_EAPT
+            )
+
+            await client_maker.gen_submit_tx_wait(
+                deposit_from_coinstore(
+                    ECONIA_ADDR, COIN_TYPE_EUSDC, market_id, 0, 600 * 10**6
+                )
+            )
+            await client_taker.gen_submit_tx_wait(
+                deposit_from_coinstore(
+                    ECONIA_ADDR, COIN_TYPE_EAPT, market_id, 0, 100 * 10**8
+                )
+            )
+
+        orders = 0
+        while base_lots_remaining > MIN_SIZE:
+            base_lots_size = random.randint(MIN_SIZE, base_lots_remaining)
+            match_price = (
+                ticks_per_lot if from_type == COIN_TYPE_EUSDC else ticks_per_lot + 1
+            )
+            volume = base_lots_size * match_price * TICK_SIZE
+            await execute_limit_order(
+                client_maker, market_id, from_type, base_lots_size, match_price
+            )
+            await execute_market_order(client_taker, market_id, to_type, base_lots_size)
+            orders += 1
+            base_lots_remaining -= base_lots_size
```

Comment:
The `volume` variable isn't used anywhere in this loop, and I don't see it used anywhere else

What is it for?

Reply:
fixed